docs/docs/recipes/integrate-logto/go-web.mdx

```diff
@@ -0,0 +1,352 @@
+---
+sidebar_label: Go Web
+---
+
+import postSignOutRedirectUriFigure from './assets/go-web-post-sign-out-redirect-uri.png';
+import redirectUriFigure from './assets/go-web-redirect-uri.png';
+import AppNote from './fragments/_app-note.mdx';
+import ConfigurePostSignOutRedirectUri from './fragments/_configure-post-sign-out-redirect-uri.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
+import GetAppSecret from './fragments/_get-app-secret.mdx';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+
+# Go Web: Integrate Logto
+
+<AppNote type="Traditional web" />
+
+## Add Logto SDK as a dependency
+
+:::tip
+We demonstrate the integration process of the SDK based on the <a href="https://gin-gonic.com">Gin Web Framework</a>. You can also easily integrate Logto into other web frameworks by taking the same steps.
+In the following code snippets, we assume your app is running on <code>http://localhost:8080</code>.
+:::
+
+Assuming your application code looks like as below:
+
+```go
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	router := gin.Default()
+	router.GET("/", func(ctx *gin.Context) {
+		ctx.String(200, "Hello Logto!")
+	})
+	router.Run(":8080")
+}
+```
+
+Execute in the project directory:
+
+```bash
+go get github.com/logto-io/go
+```
+
+Add the `github.com/logto-io/go/client` package to your application code:
+
+```go
+// main.go
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	// Add dependency
+	"github.com/logto-io/go/client"
+)
+
+func main() {
+	router := gin.Default()
+	router.GET("/", func(c *gin.Context) {
+		c.String(200, "Hello Logto!")
+	})
+	router.Run(":8080")
+}
+```
+
+## Create storage for Logto to store user credentials
+
+Logto stores user credentials in the `Storage`.  
```

Comment:
Logto stores user credentials in the `Storage`.  

This line feels odd. 
Since `Storage` here does not specify any specific storage type. Hence 'the Storage'  has no implication here.